File: model/src/test/java/no/nav/pensjon/vtp/testmodell/FiktivtNavnTest.java
package no.nav.pensjon.vtp.testmodell;

import no.nav.pensjon.vtp.testmodell.personopplysning.PersonNavn;
import no.nav.pensjon.vtp.testmodell.util.FiktivtNavn;
import org.junit.Assert;
import org.junit.Test;

public class FiktivtNavnTest {

    @Test
    public void fiktiv_navn_skal_fungere() {
        PersonNavn randomFemalName = FiktivtNavn.getRandomFemaleName();
        Assert.assertTrue(randomFemalName.getFornavn().length() > 1);
        Assert.assertTrue(randomFemalName.getEtternavn().length() > 1);

        PersonNavn randomMaleName = FiktivtNavn.getRandomMaleName();
        Assert.assertTrue(randomMaleName.getFornavn().length() > 1);
        Assert.assertTrue(randomMaleName.getEtternavn().length() > 1);
    }
}

File: model/src/test/kotlin/no/nav/pensjon/vtp/testmodell/FiktivtNavnTest.kt
package no.nav.pensjon.vtp.testmodell

import no.nav.pensjon.vtp.testmodell.kodeverk.Kjønn
import no.nav.pensjon.vtp.testmodell.util.FiktivtNavn
import org.junit.Assert
import org.junit.Test

class FiktivtNavnTest {
    @Test
    fun fiktiv_navn_skal_fungere() {
        val randomFemaleName = FiktivtNavn.getRandomName(Kjønn.K)
        Assert.assertTrue(randomFemaleName.fornavn.length > 1)
        Assert.assertTrue(randomFemaleName.etternavn.length > 1)

        val randomMaleName = FiktivtNavn.getRandomName(Kjønn.M)
        Assert.assertTrue(randomMaleName.fornavn.length > 1)
        Assert.assertTrue(randomMaleName.etternavn.length > 1)
    }
}
